libevmasm/Assembly.cpp

```diff
@@ -317,6 +317,9 @@ Json::Value Assembly::assemblyJSON(map<string, unsigned> const& _sourceIndices)
 		case PushData:
 			collection.append(createJsonValue("PUSH data", sourceIndex, i.location().start, i.location().end, toStringInHex(i.data())));
 			break;
+		case VerbatimData:
```

Comment:
Probably `VerbatimData` is technically correct from libevmasm's view, but maybe `VerbatimInstruction` or `OpaqueInstruction` is more clear in intent?

Reply:
VerbatimBytecode?

Reply:
I think it is good as VerbatimBytecode